docs/python/reference/contexts.md

```diff
@@ -206,6 +206,29 @@ def example_workflow(var1: str, var2: str):
 handle: WorkflowHandle = DBOS.start_workflow(example_workflow, "var1", "var2")
 ```
 
+### start_workflow_async
+
+```python
+DBOS.start_workflow_async(
+    func: Workflow[P, Coroutine[Any, Any, R]],
+    *args: P.args,
+    **kwargs: P.kwargs,
+) -> Coroutine[Any, Any, WorkflowHandleAsync[R]]
+```
+
+Start an asyncronous workflow in the background and return a [handle](./workflow_handles.md) to it.
```

Comment:
```suggestion
Start an asynchronous workflow in the background and return a [handle](./workflow_handles.md) to it.
```

Reply:
And in all the other places

Reply:
🤦‍♂️

OK, but first I'm re-installing https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker